bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts

```diff
@@ -79,7 +79,7 @@ export class KtbTriggerSequenceComponent implements OnInit, OnDestroy, AfterView
   public customFormData: CustomSequenceFormData = {};
 
   @Input() public projectName!: string;
-  @Input() public stage: string | undefined;
+  @Input() public stage: string | undefined | null;
```

Comment:
We actually don't need `undefined` here. With the async pipe it can either be null or string.
```ts
@Input() public stage: string | null = null
```